egs2/cmu_indic/tts1/local/clean_text.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
```

Comment:
For espnet2, text cleaning will be performed on the fly.
So you do not need it.